packages/@o3r/artifactory-tools/.gitignore

```diff
@@ -0,0 +1,6 @@
+/core
```

Comment:
I think this was generated, but it is not so accurate :D It could be adapted at least to include the `dist` directory :D